src/commands/install/extract_info_from_repo_id.py

```diff
@@ -55,16 +62,16 @@ def extract_info_from_repo_id(repo_id: str) -> PackageInfo:
     return replace(result, name=result.name.replace("-", "_").replace(".", "_"))
 
 
-def _map_name_to_url(name: str) -> Optional[str]:
+def _map_name_to_url(name: str) -> str:
     return f"https://github.com/{name}"
 
 
-def _map_ssh_to_url(ssh: str) -> Optional[str]:
+def _map_ssh_to_url(ssh: str) -> str:
     slug = _extract_slug_from_ssh(ssh)
     domain_match = re.search(r"(?<=git@).*(?=:)", ssh)
 
     if domain_match is None:
-        return None
+        raise installation_exceptions.InvalidPackageName("Couldn't map SSH to URL.")
 
     return f"https://{domain_match.group()}/{slug}"
```

Comment:
What do you think about writing tests for those functions? It can be error prone

Reply:
Ok